src/makeCalculator.js

```diff
@@ -37,7 +37,42 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: (x, obj) => {
+      obj.result += x;
+    },
+
+    subtract: (x, obj) => {
```

Comment:
I would name variables more clear than x and obj to understand what do they mean and what is happening